helpers/aggregateCart.ts

```diff
@@ -0,0 +1,16 @@
+import type { ICartItem } from "@models/carts";
+
+export const aggregateCart = (userCart: ICartItem[], product: Pick<ICartItem, "productId">) => {
+	const isInCart = userCart.some(({ productId }) => productId === product.productId);
+	let updatedCart = [] as ICartItem[];
+	if (isInCart) {
+		updatedCart = userCart.reduce((acc, current) => {
+			return current.productId === product.productId
+				? [...acc, { productId: current.productId, quantity: current.quantity + 1 }]
```

Comment:
Czy Twoje 'addToCart' nie obsluguje dodawania wielu elementów? Przydałoby sie żeby tak sie dało, bo to typowy use case w e-commerce, gdzie mozesz wybrac np. 10 sztuk i dodać do koszyka

Reply:
Zastanawiałem się czy to dodawać. Dzięki, zrobię